From 066f3b2506c6b74ca5afb980aac48fd062289555 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Mon, 13 Jun 2016 12:05:10 +0800
Subject: [PATCH 1088/5242] MLK-12900-1: ARM: dts: pin confict for spdif and
 wdog

commit  86f0086e29cc0d5a7e0c2e0d74c76478b1071e2f from
https://source.codeaurora.org/external/imx/linux-imx.git

Disable wdog in wm8958.dts for conflict with spdif

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
(cherry picked from commit 71863bcd34420384f5ef5eeecc2c8ca45389db24)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts    |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts
index fbbc648..8ff24da 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts
@@ -149,3 +149,7 @@
 &usdhc2 {
 	no-1-8-v;
 };
+
+&wdog1 {
+	status = "disabled";
+};
-- 
1.7.9.5

